src/actions/task-definitions.js

```diff
@@ -88,6 +89,38 @@ tasks.moveCellDown = new UserTask({
   },
 })
 
+tasks.highlightUp = new UserTask({
+  title: 'Highlight Cell Above',
+  displayKeybinding: 'Shift+Up', // '\u21E7 \u2191',
+  keybindings: ['shift+up'],
+  keybindingPrecondition: isCommandMode,
+  preventDefaultKeybinding: true,
+  callback() {
+    const cellAboveId = getCellAboveSelectedId()
+    dispatcher.highlightCell(getSelectedCellId(store.getState()), false)
```

Comment:
this `store.getState()` should be moved inside of the highlightCell action creator (we're trying to get better about import the redux store in as few places as possible, it makes testing more difficult)